meta/state.go

```diff
@@ -62,6 +64,16 @@ func (r *localRaft) remove() error {
 	return nil
 }
 
+func (r *localRaft) shutdown() error {
```

Comment:
No, close is mainly there to free all resources for testing.  Shutdown is for a live server that I need to stop participating in the cluster.  There is a lot of code that depends on the raftStore not being nil currently, and close will nil it out and panics galore will ensue.  Something to address in a future PR.